addons/pos_loyalty/static/src/overrides/models/pos_order.js

```diff
@@ -97,6 +97,12 @@ patch(PosOrder.prototype, {
     setupState(vals) {
         super.setupState(...arguments);
         this.uiState.disabledRewards = new Set(vals?.disabledRewards || []);
+        for (const [key, pe] of Object.entries(this.uiState.couponPointChanges)) {
+            if (!this.models["loyalty.program"].get(pe.program_id)) {
+                delete this.uiState.couponPointChanges[key];
+                continue;
+            }
+        }
```

Comment:
`continue` is useless here.

What you can do (or just remove it)


```suggestion
        for (const [key, pe] of Object.entries(this.uiState.couponPointChanges)) {
            if (this.models["loyalty.program"].get(pe.program_id)) {
                continue;
            }

            delete this.uiState.couponPointChanges[key];
        }
```